tools/docker/build_docker_util.py

```
@@ -0,0 +1,87 @@
+    machine = platform.machine().lower()
+    architecture = Architecture.UNKNOWN
+    # Check if running on Jetson (presence of jetson-specific files)
+    if os.path.exists('/etc/nv_tegra_release'):
+        # Parse JetPack version from the release file
+        try:
+            with open('/etc/nv_tegra_release', 'r') as f:
+                content = f.read()
+                # JetPack 6.x detection
+                if 'R36' in content or 'R37' in content:
+                    return 'jetpack6'
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)